### DISCUSSION TITLE
Exploring post formats in a block theme world (maybe with 6.7 additions)

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#295] [Topic Ideas]
There's been a request to include some level of post format support in the upcoming TT5 theme: https://make.wordpress.org/core/2024/07/08/what-would-you-like-to-see-in-the-next-default-wordpress-theme/#comment-46916

This is a proposal to explore various options that theme authors can do in WordPress 6.6 (and possibly more in 6.7). The post would cover some things that themers would be able to do:

- "Register" and use existing templates for archives and single posts.
- Building patterns for that would work well for specific post formats.
- Adding a custom variation to the Post Terms block for displaying a post's format.

Also, there are some related tickets to this:

- https://github.com/WordPress/twentytwentyfive/issues/17
- https://github.com/WordPress/gutenberg/pull/64167

The last ticket would add some improvements to Query Loop to make querying by post format easier. With any luck, this lands in WP 6.7.

In recent months, I've also explored what some baseline post format support would look like from a theme perspective. You can view the various PRs linked in the below ticket for some examples of what this tutorial might explore:

- https://github.com/x3p0-dev/x3p0-ideas/issues/17

At this point, this idea is very much a "work in progress," but I think it's definitely worth exploring in more depth if some support lands in TT5. I'm not 100% sold on this being a complete proposal yet, mostly because I haven't already written some version of it in my head. It's more of an inkling of an idea with a little bit of code exploration.

**Comment by bph** [comment]
This topic was approved during 9/5 Editorial group meeting. The discussion continues on the issue
- #304
